src/components/pages/Offerers/Offerer/VenueV1/VenueEdition/VenueProvidersManager/StocksProviderForm/StocksProviderForm.jsx

```diff
@@ -1,54 +1,118 @@
 import PropTypes from 'prop-types'
-import React, { useState, useCallback } from 'react'
+import React, { useState, useCallback, useRef } from 'react'
 
+import { DialogBox } from 'components/layout/DialogBox/DialogBox'
+import Icon from 'components/layout/Icon'
 import Spinner from 'components/layout/Spinner'
 
+import { ReactComponent as AlertSvg } from './assets/ico-alert-grey.svg'
+
 const StocksProviderForm = ({ createVenueProvider, providerId, siret, venueId }) => {
   const [isCheckingApi, setIsCheckingApi] = useState(false)
+  const [isConfirmDialogOpened, setIsConfirmDialogOpened] = useState(false)
+  const confirmButtonRef = useRef()
 
-  const handleFormSubmit = useCallback(
-    event => {
-      event.preventDefault()
+  const handleFormSubmit = useCallback(event => {
+    event.preventDefault()
+    setIsConfirmDialogOpened(true)
+  }, [])
 
-      setIsCheckingApi(true)
+  const abortImportingFromProvider = useCallback(() => {
+    setIsConfirmDialogOpened(false)
+  }, [])
 
-      const payload = {
-        providerId,
-        venueIdAtOfferProvider: siret,
-        venueId,
-      }
+  const confirmImportingFromProvider = useCallback(() => {
+    setIsCheckingApi(true)
 
-      createVenueProvider(payload)
-    },
-    [createVenueProvider, providerId, siret, venueId]
-  )
+    const payload = {
+      providerId,
+      venueIdAtOfferProvider: siret,
+      venueId,
+    }
+
+    createVenueProvider(payload)
+    setIsConfirmDialogOpened(false)
+  }, [createVenueProvider, providerId, siret, venueId])
 
   if (isCheckingApi) {
     return <Spinner message="Vérification de votre rattachement" />
   }
 
   return (
-    <form
-      className="stocks-provider-form"
-      onSubmit={handleFormSubmit}
-    >
-      <div className="account-section">
-        <div className="account-label">
-          {'Compte'}
+    <>
+      <form
+        className="stocks-provider-form"
+        onSubmit={handleFormSubmit}
+      >
+        <div className="account-section">
+          <div className="account-label">
+            {'Compte'}
+          </div>
+          <div className="account-value">
+            {siret}
+          </div>
         </div>
-        <div className="account-value">
-          {siret}
+        <div className="provider-import-button-container">
+          <button
+            className="secondary-button"
+            type="submit"
+          >
+            {'Importer'}
+          </button>
         </div>
-      </div>
-      <div className="provider-import-button-container">
-        <button
-          className="secondary-button"
-          type="submit"
+      </form>
+      {isConfirmDialogOpened && (
+        <DialogBox
```

Comment:
à déplacer dans un composant dédié `.../VenueEdition/VenueProviderManager/ConfirmDialog/ConfirmDialog.jsx`


```suggestion
      { isConfirmDialogOpened && (
        <ConfirmDialog 
         onConfirm={handleFormSubmit} 
         onCancel={handleCloseConfirmDialog}
        /> 
      )}
```